core-model/src/main/java/com/sequenceiq/cloudbreak/domain/Cluster.java

```diff
@@ -31,6 +14,22 @@
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
 import javax.persistence.UniqueConstraint;
+import java.util.HashSet;
```

Comment:
Can you change the import order please? I think there should be an idea settings file in the config folder.